client_test.go

```diff
@@ -4,11 +4,18 @@ import (
 	"context"
 	"net/http"
 	"net/http/httptest"
+	"os"
 	"strconv"
 	"testing"
 	"time"
 )
 
+func init() {
+	originEndpoint = os.Getenv("FIREBOLT_ENDPOINT")
```

Comment:
nit: `originalEndpoint` to make it clearer. I was confused until I've read the code.